subprojects/docs/src/samples/native-binaries/cunit/completeCUnitExample.out

```diff
@@ -0,0 +1,5 @@
+
+There were test failures:
+  1. /home/user/gradle/samples/src/operatorsTest/c/test_plus.c:6  - plus(0, -2) == -2
+  2. /home/user/gradle/samples/src/operatorsTest/c/test_plus.c:7  - plus(2, 2) == 4
+
```

Comment:
did you pick up the changes that we made to the samples test runner in `gradle/gradle` that merges stdout and stderr, so the that failure details and so on are also included in the output (and can be verified)? It looks like this is missing given this output file.

Reply:
Bummer, I did not. I'll fix this up, and hope I can merge quickly so we don't lose other changes like this.